tensorboard/plugins/graph/tf_graph/tf-graph.ts

```diff
@@ -298,12 +302,12 @@ class TfGraph extends LegacyElementMixin(PolymerElement) {
             };
           }),
         });
-        this._setRenderHierarchy(renderGraph);
-        this.async(function () {
-          this.fire('rendered');
-        });
-      }.bind(this)
+
+        return renderGraph;
+      }
     );
+
+    anyThis._setRenderHierarchy(renderGraph);
```

Comment:
Before, `_buildRenderHierarchy` was a no-op when `graphHierarchy.root.type !== tf_graph.NodeType.META`.  Now, we will end up calling `anyThis._setRenderHierarchy(undefined)`.  Is this change to logic intentional and safe?

Or does it make sense to pull our the non-meta-root-node check to the very top of `_buildRenderHierarchy` and do an early return?